core/jobs/job_options.py

```
@@ -73,6 +73,18 @@ class JobOptions(pipeline_options.PipelineOptions):  # type: ignore[misc]
 
+    # A subset of Google Cloud options related to security and networking.
+    # For the complete list of available options, refer to:
+    # https://docs.cloud.google.com/dataflow/docs/reference/pipeline-options#security_and_networking
+    GOOGLE_CLOUD_OPTIONS = {
+        'service_account_email': (
+            str,
+            'Identifies a user-managed worker service account to run virtual '
+            'machines as. If not set, workers use the Compute Engine service '
+            'account of the project as the worker service account.',
+        ),
+    }
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)